tasks/5.js

```diff
@@ -0,0 +1,10 @@
+function wordSearch(w) {
```

Comment:
The task description probably wasn't the best, but the idea was to have a function which would accept two arguments (`word` and `text`) and would work as follows:
```js
wordSearch('foo', 'foobar') // false
wordSearch('foo', 'Foo bar') // true
wordSearch('foo', ' foo bar') // true
```